siad/daemon.go

```diff
@@ -305,6 +306,16 @@ func startDaemonCmd(cmd *cobra.Command, _ []string) {
 		go profile.StartContinuousProfile(globalConfig.Siad.ProfileDir)
 	}
 
+	// Establish a loop that will clear memory every 30 seconds. This will help
+	// keep 'siad' as lightweight as possible. Can be removed once peak usage is
+	// below 1 GiB.
+	go func() {
+		for {
+			debug.FreeOSMemory()
+			time.Sleep(time.Second * 30)
+		}
```

Comment:
I believe that is correct. I was thinking about putting this even at like 10 seconds. The GC operation only takes a couple of milliseconds I believe. Mostly this is to keep the RAM usage as low as possible.